internal/pkg/log/assert.go

```diff
@@ -0,0 +1,85 @@
+package log
+
+import (
+	"bufio"
+	"strings"
+
+	"github.com/keboola/go-utils/pkg/wildcards"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/encoding/json"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
+)
+
+func CompareJSONMessages(expected string, actual string) error {
+	expectedScanner := bufio.NewScanner(strings.NewReader(strings.Trim(expected, "\n")))
+	actualScanner := bufio.NewScanner(strings.NewReader(strings.Trim(actual, "\n")))
+
+	for expectedScanner.Scan() {
+		expectedMessage := expectedScanner.Text()
+		var expectedMessageData map[string]any
+		err := json.DecodeString(expectedMessage, &expectedMessageData)
+		if err != nil {
+			return errors.Wrapf(err, "Expected string contains invalid json:\n%s", expectedMessage)
```

Comment:
Go error should start with lower case letter, see: 
https://go.dev/wiki/CodeReviewComments#error-strings

```
Error strings should not be capitalized (unless beginning with proper nouns or acronyms) or end with punctuation, since they are usually printed following other context.
```

Reply:
I see no linter for it: https://golangci-lint.run/usage/linters/

Reply:
Yes, it also seemed to me that some linter checked it in the past. 
The problem will probably be our `errors` package, and I think that the linter could not be configured.
I'll try to find it.

Reply:
```
warning: error strings should not be capitalized or end with punctuation or a newline (golint)
```

Reply:
> The repository of the linter has been archived by the owner. Replaced by revive.

Reply:
https://golangci-lint.run/usage/linters/#revive

Reply:
But I don't see that rule there: https://github.com/mgechev/revive